src/Paket/Program.fs

```diff
@@ -156,14 +162,18 @@ try
         else
             match command with
             | Command.Init -> Dependencies.Create() |> ignore
-            | Command.Add -> 
-                let packageName = results.GetResult <@ CLIArguments.Nuget @>
-                let version = 
-                    match results.TryGetResult <@ CLIArguments.Version @> with
-                    | Some x -> x
-                    | _ -> ""
-
-                Dependencies.Locate().Add(packageName, version, force, hard, interactive, noInstall |> not)
+            | Command.Add ->
+                match results.TryGetResult <@ CLIArguments.Nuget @> with
+                | Some packageName ->
+                    let version = match results.TryGetResult <@ CLIArguments.Version @> with
+                                  | Some x -> x
+                                  | _ -> ""
+
+                    Dependencies.Locate().Add(packageName, version, force, hard, interactive, noInstall |> not)
+                | None -> match results.TryGetResult <@ CLIArguments.Url @> with
```

Comment:
I think that's ok for now. We can refactor later (and put this into the Add module)